fast_plotter/plotting.py

```diff
@@ -401,10 +401,11 @@ def plot_1d_many(df, prefix="", data="data", signal=None, dataset_col="dataset",
             error = "markers"
         else:
             raise RuntimeError(err_msg)
-        if 'ratio_ylim' not in kwargs.keys():
-            kwargs['ratio_ylim'] = [0., 2.]
+        kwargs.setdefault("ratio_ylim", [0., 2.])
+        kwargs.setdefault("ratio_ylabel", "Data / MC")
         plot_ratio(summed_data, summed_sims, x=x_axis,
-                   y=y, yerr=yerr, ax=summary_ax, error=error, ylim=kwargs['ratio_ylim'])
+                   y=y, yerr=yerr, ax=summary_ax, error=error,
+                   ylim=kwargs["ratio_ylim"], ylabel=kwargs["ratio_ylabel"])
```

Comment:
No need to change, but an alternative which might be easier to read and wouldn't edit the kwargs dict would be:
```python
        ratio_ylim = kwargs.get("ratio_ylim", [0., 2.])
        ratio_ylabel = kwargs.get("ratio_ylabel", "Data / MC")
        plot_ratio(summed_data, summed_sims, x=x_axis,
                   y=y, yerr=yerr, ax=summary_ax, error=error,
                   ylim=ratio_ylim, ylabel=ratio_ylabel)
```